requirements.txt

```diff
@@ -1,3 +1,2 @@
 bpy-addon-build>=0.3.0
 fake-bpy-module-2.80==20230117
-flake8==5.0.4
```

Comment:
In contrast, I do think we should set flake8 as an expectation. I want to set it up as a github runner. even if we don't have the quota to reliably run unittests remotely, would be nice to perform automated flake8 syntax and style checks as one good method to safeguard future changes.

Reply:
Nevermind this comment - I see you have flake8 in the main config.